### PR TITLE
Feature/rails 5 comptability

### PR DESCRIPTION
**Pull request by islam-taha**
In Rails 5, `ActiveModel` Dirty API has changed in after_* callbacks, changed? should be replaced with previously_changed?.

### CODE REVIEW

File: lib/louisville/extensions/history.rb [hunk 1]
```diff
@@ -27,8 +27,8 @@ def self.included(base)
           extend ::Louisville::Extensions::History::ClassMethods
 
           # If our slug has changed we should manage the history.
-          after_save :delete_matching_historical_slug,  :if => :louisville_slug_changed?
-          after_save :generate_historical_slug,         :if => :louisville_slug_changed?
+          after_save :delete_matching_historical_slug,  if: -> { ActiveRecord.version.to_s >= '5' ? louisville_slug_previously_changed? : louisville_slug_changed? }
```
Comment (aalong-tr) [inline lib/louisville/extensions/history.rb]: If you've changed the Gemfile to require ActiveRecord 5.2.X, are the checks for ActiveRecord.version necessary?

Comment (islam-taha) [reply lib/louisville/extensions/history.rb]: yes, i'll do this.

Comment (islam-taha) [reply lib/louisville/extensions/history.rb]: @aalong-tr I've updated it now.

**Review by htanata — APPROVED** [review]
LGTM!

**Review by aalong-tr — APPROVED** [review]
Once this is merged let's go ahead and create a `v0.0.6` tag on taskrabbit:master to target in the Gemfile

**Comment by aalong-tr** [comment]
...thinking on it, I'll do that now

**Pull request merged**